src/libsyntax/ast.rs

```diff
@@ -876,7 +900,16 @@ impl Decl {
     }
 }
 
-/// represents one arm of a 'match'
+/// An arm of a 'match',
+///
+/// e.g. `0...10 => { println!("match!") }` as in
+///
+/// ```
```

Comment:
Not strictly necessary as it's the default but I like to annotate this code blocks with the language they are written in. Also, this one should probably be 'ignored' in tests, so:

    ```rust, ignore